.github/workflows/ci-benchmark.yml

```diff
@@ -44,11 +44,18 @@ jobs:
         with:
           java-version: 1.8
       - run: mvn clean install --projects janusgraph-all -Pjanusgraph-cache -Dmaven.javadoc.skip=true ${{ env.BUILD_MAVEN_OPTS }}
-
+      - run: mvn verify --projects janusgraph-all -Pjanusgraph-cache ${{ env.VERIFY_MAVEN_OPTS }}
   benchmark:
     name: Performance regression check
     runs-on: ubuntu-latest
+    needs: build-all
     steps:
+      # install CCM for CQL benchmarks
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - run: pip install git+https://github.com/li-boxuan/ccm.git
```

Comment:
I tried many many times and the CI only worked with my custom changes to `ccm`. I will submit a PR to the upstream repository.